Several fixes

artscoop opened the pull request:
Several fixes on documentation, Django 1.6 support and moderation form creation. Among various compatibility fixes, made the code compatible with django-ajax-selects.

artscoop committed:
Support Django 1.5 Custom User Model

artscoop committed:
Allow ModelAdmin form settings

artscoop committed:
i18n

artscoop committed:
add locale

artscoop committed:
+ ‘.sparkleshare’

artscoop committed:
try coping with removed fields

artscoop committed:
pass if moderated object fields are wrong

artscoop committed:
revert deserialization behaviour

artscoop committed:
Deleted src/moderation/migrations/0001_initial.py

artscoop committed:
Deleted src/moderation/migrations/__init__.py

artscoop committed:
Deleted src/moderation/migrations/__init__.py

artscoop committed:
Fix for django 1.6

Fix metaclass information, which conflicts with another metaclass introduced in Django 1.6a1; just use type.

artscoop committed:
Fix copy of original form Meta

Important fix so that moderation does not break on most custom admin forms.

artscoop committed:
Update README.rst

Minor edit on moderator use.

artscoop committed:
Update admin.py

Fix ModeratedObjectForm not following original form's definition (and therefore making the form not work as expected).

sbussetti commented:
I'm not 100% on why this pull request was made against my fork and not the primary repo, but I will certainly take a look at it, merge if appropriate and try and get it pulled upstream.

artscoop committed:
Update managers.py

Add instance check method, in case ModeratedObject table is truncated

artscoop committed:
Update register.py

On updated objects, create a ModeratedObject if it does not exist. Otherwise there is an error

artscoop committed:
Update register.py

artscoop committed:
Update helpers.py

Fix module import, in Django 1.7 some `INSTALLED_APPS` can be classes and not modules.

artscoop committed:
Update helpers.py

End fix
